hlr/testdata/hlrListObject.json

```diff
@@ -0,0 +1,34 @@
+{
+    "offset": 0,
+    "limit": 20,
+    "count": 2,
+    "totalCount": 2,
+    "links": {
+        "first": "https:\/\/rest.messagebird.com\/hlr\/?offset=0",
```

Comment:
Do we need to escape slashes here? Is this how we get it from API?

Reply:
@azmb You're right, escaping them is not needed: I'll do a quick replace!